components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -201,7 +214,12 @@ object Glean {
      * @return A [Job], **only** to be used when testing, which allows to wait on
      *         the ping submission.
      */
-    fun handleEvent(pingEvent: PingEvent): Job {
+    fun handleEvent(pingEvent: PingEvent): Job? {
+        if (!isInitialized()) {
+            logger.error("Glean must be initialized before handling events.")
+            return null
```

Comment:
I'm not sure about this, but would it be possible to do something like `return Job()`, instead of making the return type nullable?

Reply:
Yep.  Looks like that works.